ptr/ptr.go

```diff
@@ -0,0 +1,9 @@
+package ptr
```

Comment:
I'd suggest that this package isn't necessary. Just inline the functions into the relevant places.
FWIW I tend to use `newT` for a function that returns a pointer to type T:

    func newBool(b bool) *bool {
        return &b
    }

Reply:
Package has been removed.